frontend/src/Editor/LeftSidebar/SidebarDebugger.jsx

```diff
@@ -56,20 +57,41 @@ export const LeftSidebarDebugger = ({ darkMode, components, errors }) => {
         })
     },[errors])
 
+    React.useEffect(() => {
+        const unReadErrors = errorLogs.length - unReadErrorCount.read
+
+        if(open ===  false && errorLogs.length !== unReadErrorCount.read) {
+            setUnReadErrorCount((prev) => {
+                let copy = JSON.parse(JSON.stringify(prev))
+                copy.unread = unReadErrors
+                return copy
+            })
+        }
+        
+        if(open === true) {
```

Comment:
I don't think a check is required here it can also be written as `if(open)`

Reply:
Yep, we don't need this, just can put it in the else block and its all good